src/named.rs

```diff
@@ -211,3 +205,108 @@ impl std::os::windows::io::AsRawHandle for NamedTempFile {
         (**self).as_raw_handle()
     }
 }
+
+
+
+#[derive(Debug, Clone)]
+pub struct CustomNamedTempFile {
+    random_len: usize,
+    prefix: String,
+    postfix: String    
+}
+
+impl CustomNamedTempFile {
+
+    /// Start building CustomNamedTempFile. See `new` for more information.
+    pub fn start() -> Self {
```

Comment:
Yes, I know but it conflicts the name of the function creating a tempfile. AFAK Rust doesn't have overloading. I caught up some options

1. `new` to allocate the builder and `create` to create a tempfile
2. `builder` to allocate the builder and `new` to create a tempfile

Do you think the 1st option is the best way?